src/jit/compiler.hpp

```diff
@@ -3262,6 +3264,9 @@ unsigned           genMapIntRegNumToRegArgNum(regNumber regNum)
 inline
 unsigned           genMapFloatRegNumToRegArgNum(regNumber regNum)
 {
+#ifdef ARM_SOFTFP
+    return genMapIntRegNumToRegArgNum(regNum);
+#endif
     assert (genRegMask(regNum) & RBM_FLTARG_REGS);
```

Comment:
Ok, so do you mean we'll transform the values from fp regs to int arg regs anywhere else?
As I understand there's a key function for passing arguments in morp.cpp which has the following code:
nextRegNum = passUsingFloatRegs ? genMapFloatRegArgNumToRegNum(nextFltArgRegNum) : genMapIntRegArgNumToRegNum(intArgRegNum);